pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1760,19 +1760,33 @@ fn basic_fuzzing() {
             let instruction_counter = Settlement::instruction_counter();
 
             // initialize balances
-            for i in 0..10 {
-                for j in 0..4 {
-                    balances.insert((tickers[i * 4 + j], users[j].did, "init").encode(), 100_000);
+            for ticker_id in 0..10 {
+                for user_id in 0..4 {
                     balances.insert(
-                        (tickers[i * 4 + j], users[j].did, "final").encode(),
+                        (tickers[ticker_id * 4 + user_id], users[user_id].did, "init").encode(),
+                        100_000,
+                    );
+                    balances.insert(
+                        (
+                            tickers[ticker_id * 4 + user_id],
+                            users[user_id].did,
+                            "final",
+                        )
+                            .encode(),
                         100_000,
                     );
                     for k in 0..4 {
-                        if j == k {
+                        if user_id == k {
                             continue;
                         }
-                        balances.insert((tickers[i * 4 + j], users[k].did, "init").encode(), 0);
-                        balances.insert((tickers[i * 4 + j], users[k].did, "final").encode(), 0);
+                        balances.insert(
+                            (tickers[ticker_id * 4 + user_id], users[k].did, "init").encode(),
```

Comment:
and really the entire `balances.insert` save for `"init"` vs. `"final"` here and below